src/qt/guiutil.cpp

```diff
@@ -411,14 +411,22 @@ void bringToFront(QWidget* w)
 #endif
 
     if (w) {
-        // activateWindow() (sometimes) helps with keyboard focus on Windows
-        if (w->isMinimized()) {
-            w->showNormal();
-        } else {
+        if (QGuiApplication::platformName() == "wayland") {
+            auto flags = w->windowFlags();
+            w->setWindowFlags(flags|Qt::WindowStaysOnTopHint);
+            w->show();
+            w->setWindowFlags(flags);
             w->show();
+        } else {
+            // activateWindow() (sometimes) helps with keyboard focus on Windows
```

Comment:
Shouldn't macOS-specific `ForceActivation();` go in this branch as well?

Reply:
Yeah, it makes sense.